### DISCUSSION TITLE
Problem while performing tranasction on smartcontract by gnosis wallet

### DISCUSSION THREAD

**Discussion by Anup9444** [rainbow-me/rainbowkit#1431] [Q&A] [answered]
Hi,
I am getting an error while confirming the transaction by gnosis wallet ... it works perfectly fine while confirming transactions with another wallet like metamask...
the error I encountered while providing an allowance 
error:
![image](https://github.com/rainbow-me/rainbowkit/assets/110972959/25e7056d-b41e-4861-b32a-1dfea4db975f)

**Comment by DanielSinclair** [comment, marked as answer]
This is likely an issue in the Safe Apps SDK or the Safe Wallet. Try to open an issue on their repos.
